pkg/cmd/run.go

```diff
@@ -570,6 +628,10 @@ func init() {
 	runCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
 	runCmd.Flags().Bool("no-exit-on-write-failure", false, "do not exit if unable to write the fallback file")
 	runCmd.Flags().Bool("forward-signals", forwardSignals, "forward signals to the child process (defaults to false when STDOUT is a TTY)")
+	// secrets mount flags
+	runCmd.Flags().String("mount", "", "write secrets to an ephemeral file, accessible at DOPPLER_CLI_SECRETS_PATH. when enabled, secrets are NOT injected into the environment")
```

Comment:
Might be worth stating that the CLI automatically sets the format if a file extension is provided as well as calling out the name transformers are supported would be beneficial as they will negate the need for the `--mount-format` option in nearly all cases.

```suggestion
	runCmd.Flags().String("mount", "", mount ephemeral secrets file in format set by file extension (JSON by default). supports --name-transformers. no environment variables injected except $DOPPLER_MOUNT_PATH.")
```